subscription/upgrade.md

```diff
@@ -9,92 +9,71 @@ redirect_from:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
 2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The **Billing Details** page displays billing information about your existing user account.
 
     >**Note**
     >
-    > Ensure you have selected your personal account or organization on the
+    > Ensure you have selected the correct account on the
     left side of the **Billing Details** page. For example, if you want to upgrade to a Docker Pro subscription, you must select your personal account.
 
-    ![Billing details personal account](images/billing-personal-account.png){:width="750px"}
-
-3. Click **Change plan** and navigate the pricing page to upgrade your subscription and continue with the on-screen instructions.
+3. Click **Change plan** and navigate the pricing page to upgrade your subscription.
 
-    You can select from the following options: Pro, Team, or Business subscription.
+4. Select your subscription and continue with the on-screen instructions.
 
-4. When upgrading to a Team subscription, provide the organization details and click **Continue to Org Size**. Review your Team subscription and click **Continue to Payment**.
-
-     > **Note**
+     >**Note**
      >
-     > You can also specify the number of users you’d like to add to your organization. On the **Organization size** page, the cost for the annual subscription is set by default. If you prefer to change this, select **Pay Monthly**.
-
-5. You’ll also receive an email confirming your new subscription.
+     > Click **Create Organization** if you want to add another organization to your subscription. Click Buy Now to proceed with the on-screen instructions.
 
-6. Open the drop-down menu next to your username in the top-right corner and select **Billing**.
+     ![Billing personal account](images/Billing-personal-account.png)
 
-    You can view the information about your new subscription and the total cost
-  on the **Billing** page.
+5. On the **Organization Size** page, select your annual or monthly subscription and update the number of users for your organization. For a Business subscription, the only option available is annually.
 
-## Upgrade to Docker Business
+> **Note**
+> To upgrade to a Business subscription, 5 is the maximum number of users you can add using a credit card and 40 users for purchase orders. If you want to create a purchase order, update the field for the users and click **Contact Sales** to complete the contact sales form.  Then, continue the on-screen instructions for the billing details and click **Purchase**.
 
-To upgrade to a Docker Business subscription, [Contact Sales](https://www.docker.com/pricing/contact-sales){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
+6. Click **Continue to Payment** and continue to on-screen instructions and click Purchase to complete your subscription.
```

Comment:
This appears at 1. The note above needs indenting . See 
![image](https://user-images.githubusercontent.com/47779042/141800030-67f5c0fa-cbc2-4a98-b7ff-64fa96e1b1e2.png)